phone-book/src/keypad.js

```diff
@@ -0,0 +1,308 @@
+const kuivstarIdentification = [
+    '067',
+    '096',
+    '097',
+    '098',
+    '068'
+];
+
+const vodafoneIdentification = [
+    '050',
+    '066',
+    '095',
+    '099'
+];
+
+const lifeIdentification = [
+    '063',
+    '093',
+    '073'
+];
+
+//phoneNumber.replace(/(.{3})(.{2})(.{2})/g, '($1) $2-$3-')
+/*difinePhoneOperator() {
+        if(kuivstarIdentification.some(phoneCode => numberField.value.includes(phoneCode))) {
+            return ` Kuivstar `;
+        }
+        
+        if(vodafoneIdentification.some(phoneCode => numberField.value.includes(phoneCode))) {
+            return ` Vodafone `;
+        }
+        
+        if(lifeIdentification.some(phoneCode => numberField.value.includes(phoneCode))) {
+            return ` Life :) `;
+        }
+        
+        return ` unknown `;
+    }
+    */
+
+const callModeButtons = [
+    {
+        className: 'fa-microphone',
+        description: 'mute'
+    },
+    {
+        className: 'fa-th',
+        description: 'keypad'
+    },
+    {
+        className: 'fa-volume-up',
+        description: 'speaker'
+    },
+    {
+        className: 'fa-plus',
+        description: 'add call'
+    },
+    {
+        className: 'fa-video',
+        description: 'video'
+    },
+    {
+        className: 'fa-users',
+        description: 'contacts'
+    },
+];    
+
+class CallMode{
+    constructor() {
+        this.callModeTemplate = `
+            <div id="call-mode" class="contact-block call-mode">
+
+                ${this.callTargetComponent()}
+
+                ${this.callAnimationComponent()}
+
+                ${this.callModeKeypad()}
+
+            </div>
+        `;
+    }
+
+    callTargetComponent() {
+        return `
+            <div class="row">
+                <span class="call-target">(098) 76-54-321</span>
+                <span class="phone-code">(Kuivstar)</span>
+            </div>
+        `
+    }
+
+    callAnimationComponent() {
+        return `
+            <div class="row">
+                <span id="load-point-1" class="fas fa-circle load-point"></span>
+                <span id="load-point-2" class="fas fa-circle load-point"></span>
+                <span id="load-point-3" class="fas fa-circle load-point"></span>
+                <span id="load-point-4" class="fas fa-circle load-point"></span>
+                <span id="load-point-5" class="fas fa-circle load-point"></span>
+            </div>
+        `
+    }
+
+    callModeKeypad() {
+        const callModeKeypadTemplate = document.createElement('main');
+        callModeKeypadTemplate.classList.add('container');
+        callModeKeypadTemplate.classList.add('call-mode-keypad-block');
+
+        const row = document.createElement('div');
+        row.classList.add('row');
+
+        let counter = 0;
```

Comment:
what does counter refers to? name convention and context of usage such variable not so clear